src/data/cosmos_chains.json

```diff
@@ -272,6 +266,12 @@
       "network_id": "sei-2",
       "image": "/images/chains/sei.svg"
     },
+    {
+      "id": "teritori",
+      "name": "Teritori",
+      "network_id": "teritori",
+      "image": "/images/chains/teritori.svg"
```

Comment:
This image doesn't exist

![image](https://github.com/axelarnetwork/axelar-docs/assets/3818101/630b5989-398b-4c08-8215-6f7af421664e)